test/amqp-integration-test.py

```diff
@@ -85,8 +87,17 @@ def run_node_test():
         print("\nIssuing failed")
         die(ExitStatus.NodeFailure)
 
+    # As OCSP-Updater is generating responses indepedantly of the CA we sit in a loop
+    # checking OCSP until we either see a good response or we timeout (5s).
     ee_ocsp_url = "http://localhost:4002"
     issuer_ocsp_url = "http://localhost:4003"
+    timeout = time.time() + 5
```

Comment:
You could move the `verify_ocsp_good` for the issuer OCSP ahead of this code, since it doesn't need to wait, and running it here gives the updater just a teensy bit more time to do what it needs.